default_values:
  tool_offset: 0.23
  agv_gain_real: 5.0
  ur_gain_real: 5.0
  agv_gain_sim: 6.0
  ur_gain_sim: 6.0
  filter_translation_resolution: 0.04  # m
  filter_orientation_resolution: 0.08  # rad
  agv_movement_delta: 0.1      # m
  min_distance_to_obs: 0.1      # m